aten/src/ATen/native/quantized/cpu/qnnpack/include/conv_utils.h

```diff
@@ -190,5 +211,160 @@ struct conv_param_t {
       ukernel_type = pytorch_qnnp_ukernel_type_conv;
     }
   }
+
+  /**
+   * @brief Computes the output dimensions given a 2D input.
+   */
+  std::array<size_t, 2> compute_output_dims(
```

Comment:
See the comment above for similar function in PT.

Reply:
The function in PT doesn't have the deconv shape